routes/database.rb

```diff
@@ -101,25 +109,30 @@ def self.registered(app)
         params[:name] = nil if params[:name] == ''
         params[:type] = nil if params[:type] == ''
         params[:server] = nil if params[:server] == ''
-        params[:active] = nil if params[:description] == ''
+        params[:active] = nil if params[:active] == ''
 
         # recieve an updated database
         db = DbResource[id: params[:id]]
+        project_db = DbResourcesProject.filter(db_resource_id: db.id).first
 
         db.update(project_id:  params[:project_id] || db.project_id,
                   name:       params[:name] || db.name,
                   type:       params[:type] || db.type,
                   server:    params[:server] || db.server,
                   modified:   DateTime.now || db.modified,
                   active: params[:active] || db.active)
+        project_db.update(project_id: params[:project_id] || project_db.project_id,
+                          db_resource_id: db.id || project_db.db_resource_id,
+                          sku_id: params[:sku_id] || project_db.sku_id)
 
         redirect "/db/#{params[:id]}"
       end
 
       app.delete '/dbs/:id/?' do
         # delete a database
         db = DbResource[id: params[:id]]
-        db.update(active: false)
+        # disassociate this db sku's and deactivate the db
```

Comment:
I think maybe reassign_resources is not the right method name for this - reassign_resources is really just for when we delete projects and clients, so their resources get reassigned back to default. This is really a delete method that also removes the project and sku associations, which is a little different and could probably just be put in the ``delete`` method of the model. This may just be a naming issue, but it confused me when reviewing the reassign_resources methods in the models.